dev/io.openliberty.data.internal_fat_datastore/build.gradle

```diff
@@ -10,5 +10,21 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
+ 
+task addDDL(type: Copy) {
+  mustRunAfter jar
+  from new File(project.projectDir, 'test-resources/')
+  into new File(autoFvtDir, 'test-resources/')
+}
 
+addRequiredLibraries.dependsOn addDDL
 addRequiredLibraries.dependsOn addDerby
+
+// Copy bundle io.openliberty.org.testcontainers to AutoFVT/lib/
+// This bundle is required both at compiletime and runtime.
+addRequiredLibraries.dependsOn copyTestContainers
+
+// If you are preforming database testing use the copyJdbcDrivers
+// task to copy the common JDBC drivers into the
+// publish/shared/resources/jdbc directory
+addRequiredLibraries.dependsOn copyJdbcDrivers
```

Comment:
The plan that Tracy and I had come up with (which was probably never discussed since transferring the work) was that fat_datastore was supposed to remain Derby only, and fat_ddlgen was supposed to be made to run against the various databases.

Reply:
If I recall, the most important difference between the two FATs is that fat_dlgen was supposed to actually execute the DDL to make sure it runs successfully against each database type.  And then the repositories all functioned properly using the generated DDL (and not automatic table creation).  On the other hand fat_datastore basically just checks that the correct table names are in the DDL.  That is because fat_datastore is testing the various DDL file names and combinations of repositories in the same file; whereas fat_ddlgen is focused on the contents of the files, which should include not just create statements, but also updates for constraints.

However, I have no issue with this test also being in the database rotation, so long as we don't loose sight of the need for fat_ddlgen to test more complex tables and actually executing the ddl.